datalad/support/annexrepo.py

```diff
@@ -197,6 +197,13 @@ def __init__(self, path, url=None, runner=None,
 
         if version is None:
             version = self.config.get("datalad.repo.version", None)
+            # we might get an empty string here
+            # TODO: if we use obtain() instead, we get an error complaining
+            # '' cannot be converted to int (via Constraint as defined for
+            # "datalad.repo.version" in common_cfg
+            # => Allow conversion to result in None?
+            if not version:
+                version = None
```

Comment:
Agree with @mih, empty is empty, not None generally. But for version I guess it would be ok

Reply:
Yeah - see #1842